mail_tracking_mailgun/models/mail_tracking_email.py

```diff
@@ -78,8 +78,8 @@ def _mailgun_values(self):
         if not api_key:
             raise ValidationError(_("There is no Mailgun API key!"))
         api_url = icp.get_param("mailgun.api_url", "https://api.mailgun.net/v3")
-        catchall_domain = icp.get_param("mail.catchall.domain")
-        domain = icp.get_param("mailgun.domain", catchall_domain)
+        catchall_domain = self.env["mail.alias.domain"].search([], limit=1).name
```

Comment:
Shouldn't this be adapted for all the possible domains?

Reply:
Well, this is done just to get a default value if an specific one isn't set in the settings.